docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2.2'
+services:
+    cli:
+        build: ./
+        volumes:
+            - ./:/data
+        working_dir: /data
+        command: ["true"]
+
+networks:
```

Comment:
Why specify the `networks` section? I don't think it's generally needed?

Reply:
Just cut-and-paste, so I tweaked. I don't think it is needed. I'll try without.